aws/policy/paas.yaml

```diff
@@ -24,6 +24,14 @@ Statement:
   - Sid: AllowResourceRestrictedActionsWhichIncurNoFees
     Effect: Allow
     Action:
+      - cloudfront:CreateDistribution
+      - cloudfront:CreateDistributionWithTags
```

Comment:
I can't find this action on https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazoncloudfront.html
Should it be `CreateStreamingDistributionWithTags`?  We'll need a `'arn:aws:cloudfront::{{ aws_account_id }}:streaming-distribution/*'` resource for the streaming distro policies.

Reply:
I added the permissions for the streaming distribution.
I was trying to add permission for this https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_CreateDistributionWithTags.html, it is strange that it is not part of the documentation